app/assets/scripts/views/countries.js

```diff
@@ -659,13 +736,70 @@ class AdminArea extends SFPComponent {
     );
   }
 
+  syncBodyOverflow = (shouldOverflow) => {
+    if (shouldOverflow) {
+      document.getElementsByTagName('html')[0].style.overflow = 'hidden';
```

Comment:
@frozenhelium is it possible to say a bit why this was needed / link to the CSS that made this required to do? Ideally we should never need to set the `overflow` of the top level HTML element to `hidden`, is there a better way we could be doing this?

Reply:
It was required because the project form is overlaid on top of countries page, and the overflow from countries page was still visible.

Reply:
This function should have been memoized

Reply:
The better way would be to implement overflow in the country page itself and hide the overflow (if still required) from the css when the project form is shown. This can be achieved as

Reply:
Ah! Let me try looking at this a bit. If I understand correctly, this is to be able to keep the overlay on top of the country page and prevent the page from scrolling? Hmm.. and `position:fixed` on the overlay does not work because it can get taller than the height of the screen?

Let's chat about this tomorrow and let me try and play around with it, but, right, I can see this being a bit tricky.